src/async_impl/client.rs

```diff
@@ -412,12 +412,6 @@ impl Client {
             reusable
         });
 
-        if proxy::is_proxied(&self.inner.proxies, &url) {
-            if uri.scheme() == Some("http") {
-                req.set_proxy(true);
```

Comment:
This is needed when not tunneling, in order to sent a `GET http://hyper.rs/guides HTTP/1.1` line to the HTTP proxy.

Reply:
Added back along with updating with proxy specific headers.